### PR TITLE
Added Prettier module

### PR DESCRIPTION
**Pull request by jonathan-raabe**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "tabWidth": 4,
+    "singleQuote": false,
+    "trailingComma": "none",
+    "overrides": [
```
Comment (JonathanLefebvreV) [inline .prettierrc:5]: May you add another rule for SCSS file doing this think of format
```
    &__panel {
        position: relative;
        height: 2rem;
        text-align: left;
        width: 100%;
        cursor: pointer;
    }
```
`:` should have no space before, one space after
`;` always finish a line

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        {
+            "files": ["*.twig", "*.html"],
+            "options": {
+                "tabWidth": 2,
+                "editorconfig": true
+            }
```
Comment (JonathanLefebvreV) [inline .prettierrc:11]: Can you add a rule to set in `.ts` and `.mdx`, than manage import
This
```
import { moduleMetadata }                       from '@storybook/angular';
import { ArgsTable, Canvas, Meta, Story }       from '@storybook/addon-docs';
```
Become
```
import { moduleMetadata } from '@storybook/angular';
import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
```

File: .prettierrc (hunk 1 above)
Comment (jonathan-raabe) [reply .prettierrc:5]: It is already working.

File: .prettierrc (hunk 2 above)
Comment (jonathan-raabe) [reply .prettierrc:11]: There is only one option to ignore special line now, We need to use like this for this. https://prettier.io/docs/en/ignore.html

**Review by JonathanLefebvreV — APPROVED** [review]

**Pull request merged**